enclave_manager/Dockerfile

```diff
@@ -165,65 +153,112 @@ RUN ldconfig \
  && rm -rf /tmp/intel-sgx-ssl \
  && echo "SGX_SSL=/opt/intel/sgxssl" >> /etc/environment
 
-COPY . /project/TrustedComputeFramework
+COPY . /project/avalon
 
 ARG SGX_MODE
 ARG MAKECLEAN
 ARG TCF_DEBUG_BUILD
-ARG DISPLAY
-ARG XAUTHORITY
-
-#TODO need to be removed later
-#install avalon-client sdk python package.
-RUN cd /project/TrustedComputeFramework/sdk \
-   && python setup.py bdist_wheel \
-   && pip3 install dist/*.whl
-
-# Install fabric python sdk
-RUN pip3 install virtualenv \
-   && pip3 install git+https://github.com/hyperledger/fabric-sdk-py.git
 
 # Environment setup
-
```

Comment:
Are these proxy settings no more needed for enclave manager. Will it work in case of h/w mode if enclave manger needs to contact IAS server ?

Reply:
Yes, its not needed, because IAS server proxy will be picked from configuration file.